diesel_cli/src/main.rs

```diff
@@ -136,12 +136,20 @@ fn create_migrations_dir(matches: &ArgMatches) -> Result<PathBuf, crate::errors:
     Ok(dir)
 }
 
-fn create_config_file(matches: &ArgMatches) -> Result<(), crate::errors::Error> {
+fn create_config_file(
+    matches: &ArgMatches,
+    migrations_dir: &Path,
+) -> Result<(), crate::errors::Error> {
     use std::io::Write;
     let path = Config::file_path(matches);
     if !path.exists() {
+        let source_content = include_str!("default_files/diesel.toml").to_string();
+        let modified_content = source_content.replace(
+            "dir = \"migrations\"",
+            &format!("dir = \"{}\"", migrations_dir.display()),
```

Comment:
I'm not sure if that works on windows, as the relevant tests seem to fail.

Reply:
Oh you are right, let me setup a Windows virtual machine so I can fix that.